Track 2 Mgmt Preview docs and readme update based on new namespace (#12871)
Azure/azure-sdk-for-net: doc/mgmt_preview_quickstart.md

    button.

Linux-based OS :

    export AZURE_CLIENT_ID="__CLIENT_ID__"
    export AZURE_CLIENT_SECRET="__CLIENT_SECRET__"
    export AZURE_TENANT_ID="__TENANT_ID__"
    export AZURE_SUBSCRIPTION_ID="__SUBSCRIPTION_ID__"

Authentication and Creating Resource Management Client
------------------------------------------------------

Now that the environment is setup, all you need to do is to create an
authenticated client. Our default option is to use
**DefaultAzureCredential** and in this guide we have picked
**Resources** as our target service, but you can set it up similarly for
any other service that you are using. **For example, in order to manage Compute or Network resources, you would create a ``ComputeManagementClient`` or ``NetworkManagementClient``**

To authenticate to Azure and create a management client, simply do the
following:
```csharp
    using Azure.Identity;
    using Azure.ResourceManager.Resources;
    using Azure.ResourceManager.Resources.Models;
    using System;
    ...
    var subscriptionId = Environment.GetEnvironmentVariable("AZURE_SUBSCRIPTION_ID");
    var resourceClient = new ResourcesManagementClient(subscriptionId, new DefaultAzureCredential());

>     var resourceGroupsClient = resourceClient.GetResourceGroupsClient();

Reviewer: ```suggestion
    var resourceGroupsClient = resourceClient.ResourceGroups;
```

```

> From this code snippet, we showed that in order to interact with Resources, we need to create a top-level client first (**ResourcesManagementClient**), then get the corresponding sub-resource client we are interested in, in this case we called **.GetResourceGroupsClient()** to get a ResourceGroupsClient

Reviewer: ```suggestion
From this code snippet, we showed that in order to interact with Resources, we need to create a top-level client first (**ResourcesManagementClient**), then get the corresponding sub-resource client we are interested in, in this case we called **.ResourceGroups** to get a ResourceGroupsOperations
```

```

Example: Creating a Virtual Machine
-----------------------------------
Let's show a concrete example of how you would create a virtual machine using .NET SDK
```csharp
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

using Azure.Identity;
using Azure.ResourceManager.Compute;
using Azure.ResourceManager.Compute.Models;
using Azure.ResourceManager.Network;
using Azure.ResourceManager.Network.Models;
using Azure.ResourceManager.Resources;
using Azure.ResourceManager.Resources.Models;

namespace AzureCreateVMSample
{
    /// <summary>
    /// Create a Virtual Machine
    /// </summary>
    public class CreateVMSample
    {
        public static async Task CreateVmAsync(
            string subscriptionId,
            string resourceGroupName,
            string location,
            string vmName)
        {
            var computeClient = new ComputeManagementClient(subscriptionId, new DefaultAzureCredential());
            var networkClient = new NetworkManagementClient(subscriptionId, new DefaultAzureCredential());
            var resourcesClient = new ResourcesManagementClient(subscriptionId, new DefaultAzureCredential());
 

>             var virtualNetworksClient = networkClient.GetVirtualNetworksClient();
>             var networkInterfaceClient = networkClient.GetNetworkInterfacesClient();
>             var publicIpAddressClient = networkClient.GetPublicIPAddresssesClient();
>             var availabilitySetsClient = computeClient.GetAvailabilitySetsClient();
>             var virtualMachinesClient = computeClient.GetVirtualMachinesClient();
>             var resourceGroupClient = resourcesClient..GetResourceGroupsClient();

Reviewer: ```suggestion
            var virtualNetworksClient = networkClient.VirtualNetworks;
            var networkInterfaceClient = networkClient.NetworkInterfaces;
            var publicIpAddressClient = networkClient.PublicIPAddressses;
            var availabilitySetsClient = computeClient.AvailabilitySets;
            var virtualMachinesClient = computeClient.VirtualMachines;
            var resourceGroupClient = resourcesClient.ResourceGroups;
```
